cli/crates/server/src/bridge/udf.rs

```diff
@@ -211,12 +211,24 @@ impl UdfRuntime {
                     futures_util::future::ready(Ok(port))
                 });
             pin_mut!(filtered_lines_stream);
-            filtered_lines_stream.try_next().await.ok().flatten().ok_or_else(|| {
-                UdfBuildError::BunSpawnFailedWithOutput {
-                    output: lines_skipped_over.join("\n"),
-                }
-            })?
+            filtered_lines_stream
+                .try_next()
+                .await
+                .ok()
+                .flatten()
+                .ok_or(lines_skipped_over)
+        };
+        let bound_port = match bound_port {
+            Ok(port) => port,
+            Err(skipped_over_lines) => {
+                let outcome = bun.wait_with_output().await.unwrap();
```

Comment:
I currently kept this as `unwrap` to match stdout but we should turn these into `expect`

Reply:
This isn't `map_err` due to the `await` here